### DISCUSSION TITLE
I cant Reverify academic affiliation github

### DISCUSSION THREAD

**Discussion by slohy99** [community/community#19619] [Projects and Issues]
when I try Reverify my academic affiliation in github education I got this error message : There are outstanding issues with your GitHub account. Please contact [GitHub Support]) for more information.
I try every think but doesn’t work what’s the problem help me please

**Comment by melvincwng** [comment]
Hi @slohy99 👋🏻, as the error message has suggested, please contact [GitHub Support](https://support.github.com/contact/account) over here.

Simply describe your issue over there, raise a ticket, and patiently wait for a couple of days for the GitHub Support team to get back to you on this.

Hope this helps you.

**Reply by slohy99** [reply]
I do that but they say same response ?

**Comment by sajibAdhi** [comment]
Hello @slohy99 
Is your problem fixed?

**Reply by slohy99** [reply]
no

**Reply by sajibAdhi** [reply]
I am also facing the same Issue.
